project-resources/community-health-files-guide.md

```diff
@@ -0,0 +1,61 @@
+# Guide to Community Health Files for OpenJS Foundation Projects
+
+Community health files are documents related to project collaboration. Most commonly, these include:
+
+- `CODE_OF_CONDUCT.md`
+- `CONTRIBUTING.md`
+- `GOVERNANCE.md`
+- `SECURITY.md`
+
+> [!NOTE]
+> See the [full list](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types) of supported default community health files
+
+## Default Community Health Files
+
+Instead of adding these files to every repository within an organization, we can add them only once to a special organization repository which must be named `.github`. GitHub will automatically use the community health files specified at this location by default for all repositories within the organization.
+
+The filenames must match [what is expected by GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types) and must be in one of these locations:
+
+- The root of the `.github` repository
+- The `.github` folder in the `.github` repository
+- The `docs` folder in the `.github` repository
+
+Where possible, it is best to have only one file for each type of community health file, specified in the `.github` repository, and to avoid having individual files for each repository. However, specific project needs may require repository-specific files.
+
+> [!Important]
+> If a file also exists in a repository, then that file will take precedence over the organization default.
+
+### Community Health Files That Projects Should Have
+
+- Organizations/projects must have at least `CODE_OF_CONDUCT.md` and `SECURITY.md` files.
+- A [`CONTRIBUTING.md`](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) file is highly recommended for all organizations/projects.
+- A `GOVERNANCE.md` file should be provided for all organizations/projects with formal governance.
+
+> [!NOTE]
+> All projects must have a LICENSE file, but it is not possible to provide a default LICENSE file for the organization. Every repository requires its own LICENSE file.
```

Comment:
```suggestion
> All projects must have a LICENSE file, but it is not possible to provide a default LICENSE file for the organization. Every repository requires its own LICENSE file. This file __must__ be included in any released artifact. 
```

Reply:
This is a good point, but this document isn't about `LICENSE` files at all; this note is included so people don't make the mistake of thinking they can specify an org-wide license to apply to all repos.  Requirements for IP, licensing, bundling, etc. are outside the scope of this document.